src/oop/programs_with_interesting_classes/easy/q9/TestCarbonFootprint.java

```diff
@@ -0,0 +1,19 @@
+package oop.programs_with_interesting_classes.easy.q9;
+
+public class TestCarbonFootprint {
+    public static void main(String[] args) {
+        Activity a1 = new Activity("Car", 30.5);
+        Activity a2 = new Activity("Bicycle", 10);
```

Comment:
this is pretty nitpicky but change this so that buses have higher emissions than bicycles